LUIS/Authoring SDK/.gitignore

```diff
@@ -0,0 +1,257 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
@mohabghanem, you missed one, the folders under LUIS are still Authoring SDK and Runtime SDK.
But its really not an issue, so up to you.

Reply:
BTW I'm not the admin of this repo, still need admin to trigger the merge.

Reply:
@yangyuan  sorry I forgot about those, they are edited now.